InternalDocs/exception_handling.md

```diff
@@ -182,3 +183,12 @@ The interpreter's function to lookup the table by instruction offset is
 The Python function ``_parse_exception_table()`` in
 [Lib/dis.py](https://github.com/python/cpython/blob/main/Lib/dis.py)
 returns the exception table content as a list of namedtuple instances.
+
+Exception Chaining Implementation
+---------------------------------
+
+[Exception chaining](https://docs.python.org/dev/tutorial/errors.html#exception-chaining)
+refers to setting the ``__context__`` and ``__cause__`` fields of an exception as it is
+being raised. The ``__context__`` field is set by ``_PyErr_SetObject()`` in
+:cpy-file:`Python/errors.c` (which is ultimately called by all ``PyErr_Set*()`` functions).
```

Comment:
(`cpy-file` is a Sphinx role; but this is now merged so you can change it next time)

Reply:
Oh oops, I missed that -- PR in a moment.

A